lib/tasks/voight_kampff.rake

```diff
@@ -8,7 +8,7 @@ namespace :voight_kampff do
     uri = URI(args[:url])
     contents = Net::HTTP.get(uri)
 
-    if contents.present?
+    if !contents.nil? && !contents.empty?
```

Comment:
Aren't `present?` and this line functionally equivalent? Why do we prefer having it in this expanded form? I just read through https://stackify.com/understanding-absence-in-ruby-present-blank-nil-empty/ to try and understand this better.

Reply:
.present? is monkey-patched onto Object by Rails and isn't pure Ruby.

I don't think this library should take a direct Rails dependency.

Reply:
https://api.rubyonrails.org/classes/Object.html#method-i-present-3F

Reply:
Gotcha gotcha